nhrpd/netlink_arp.c

```diff
@@ -154,6 +154,10 @@ int nhrp_neighbor_operation(ZAPI_CALLBACK_ARGS)
 	struct zapi_neigh_ip api = {};
 
 	zclient_neigh_ip_decode(zclient->ibuf, &api);
+
```

Comment:
Perhaps a comment on why you are allowing only /32 is welcome.

Reply:
lol.  That is what the original code did.  I just moved it from Zebra to up into nhrpd.  I don't fully understand it but if I had to guess it's a RTM_NEIGH entry where the lladr is set to a v4 address.  I left it alone mainly because I didn't know and didn't want to take the time to figure it out.